tests/lib/cli.js

```diff
@@ -1160,4 +1161,29 @@ describe("cli", () => {
         });
     });
 
+    describe("testing the cloned config", () => {
+        describe("config file and input file", () => {
+            it("should execute without any errors", async () => {
```

Comment:
Can we change this `it` description to describe what we're actually testing (i.e. "should not modify original configuration object")?

Reply:
sure 👍